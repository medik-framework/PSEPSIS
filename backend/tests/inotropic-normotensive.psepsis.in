[
  {
    "id": "tabletApp",
    "action": "broadcast",
    "eventName": "AgeEntered",
    "eventArgs": [60],
    "waitFor":
     {
       "name": "StartGUI"
     }
  },
  {
    "id": "tabletApp",
    "action": "broadcast",
    "eventName": "StartInotropicTherapy",
    "eventArgs": [],
    "waitFor" : {
      "name": "ConfirmAgeEntered"
    }
  },
  {
    "id": "tabletApp",
    "action": "broadcast",
    "eventName": "ConfirmBPObtained",
    "eventArgs": [],
    "waitFor" : {
      "name": "Instruct",
      "args": ["get BP"]
    }
  },
  {
    "id": "tabletApp",
    "action": "broadcast",
    "eventName": "ConfirmShockType",
    "eventArgs": ["Normotensive"],
    "waitFor" : {
      "name": "Recommend",
      "args": [
        "get shock type",
        "Recommendation: Normotensive management"
      ]
    }
  },
  {
    "action": "exit",
    "waitFor" : {
      "name": "RecommendDrug"
    }
  }
]
