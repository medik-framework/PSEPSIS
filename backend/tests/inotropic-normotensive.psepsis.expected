[
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "ConfirmAgeEntered",
    "args": []
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "Instruct",
    "args": ["get BP"]
  },
  {
    "id": "datastore",
    "interface": "Datastore",
    "name": "Obtain",
    "args": [
      "BP Sys"
    ]
  },
  {
    "id": "datastore",
    "interface": "Datastore",
    "name": "Obtain",
    "args": [
      "MAP"
    ]
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "Recommend",
    "args": [
      "get shock type",
      "Recommendation: Normotensive management"
    ]
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "RecommendDrug",
    "args": [
      "Dopamine",
      "Inotropic Support",
      2,
      "administer 2-20 mcg/kg/min IV/IO infusion"
    ]
  }
]
